pkg/operator/storageclasscontroller/storageclasscontroller.go

```diff
@@ -91,24 +92,31 @@ func NewStorageClassController(
 		scStateEvaluator:     evaluator,
 		backoff:              defaultBackoff,
 		nextCheck:            time.Now(),
+		connPolicyNames:      make(map[string]string),
 	}
 
 	return c
 }
 
-func (c *StorageClassController) Sync(ctx context.Context, connection []*vclib.VSphereConnection, apiDeps checks.KubeAPIInterface) error {
+func (c *StorageClassController) Sync(ctx context.Context, connections []*vclib.VSphereConnection, apiDeps checks.KubeAPIInterface) error {
 	checkResultFunc := func() (checks.ClusterCheckResult, checks.ClusterCheckStatus) {
 		sc := resourceread.ReadStorageClassV1OrDie(c.manifest)
 		scState := c.scStateEvaluator.GetStorageClassState(sc.Provisioner)
 
-		policyName, syncResult := c.syncStoragePolicy(ctx, connection, apiDeps, scState)
-		if syncResult.CheckError != nil {
-			klog.Errorf("error syncing storage policy: %v", syncResult.Reason)
-			clusterCondition := "storage_class_sync_failed"
-			utils.InstallErrorMetric.WithLabelValues(string(syncResult.CheckStatus), clusterCondition).Set(1)
-			return syncResult, checks.ClusterCheckAllGood
+		// TODO: Iterate through each vcenter for storage policy.
```

Comment:
does this todo need to be addressed?

Reply:
I believe I left this as a talking point with @gnufied when it gets reviewed.  I had some questions on "default" storage and such.  I think we really just want to map from infrastructure resource.  I'll leave this one for now and be sure to address before merge.